src/app/core/about.component.ts

```diff
@@ -23,9 +26,12 @@ export class AboutComponent implements OnInit {
 	constructor(private configService: ConfigService) {}
 
 	ngOnInit() {
-		this.configService.getConfig().subscribe((config: Config) => {
-			this.appTitle = config?.app?.title ?? '';
-			this.version = config.version;
-		});
+		this.configService
+			.getConfig()
+			.pipe(first(), untilDestroyed(this))
```

Comment:
I believe that `first()` (and `take(1)`) should generally unsubscribe once the first observable has been emitted, so untilDestroyed may be redundant here.